src/realm/util/aes_cryptor.hpp

```diff
@@ -29,8 +30,13 @@
 #elif !defined(_WIN32)
 #include <openssl/aes.h>
 #include <openssl/sha.h>
-#else
-#error Encryption is not yet implemented for this platform.
+#endif
+
+#ifdef _WIN32
+#include <windows.h>
+#include <stdio.h>
+#include <bcrypt.h>
+#pragma comment(lib, "bcrypt.lib")
```

Comment:
Please use a consistent pattern like this:
```C++
#if REALM_PLATFORM_APPLE
...
#elif defined(_WIN32)
...
#else
    // POSIX
...
#endif
```
instead of multiple blocks of preprocessor code.